<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downSamplingWindow</class>
 <widget class="QMainWindow" name="downSamplingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownSampling</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonQuit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberDecimate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>value of Density</string>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderDecimate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>change opacity of selected file</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelDecimate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimate: (%)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetLoadedMesh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
